extra/grafana/README.md

```diff
@@ -74,7 +73,7 @@ svc-readonly  local  a12345bc6-d78e-901f-23a4-56b07b89012  2022-11-30 08:58:40 E
 ```
 
 3. Configure `/etc/prometheus/prometheus.yaml` to point use the OpenMetrics exporter to query the device endpoint.
-An example of a single FlashArray device configuration is here: [](../prometheus/prometheus.yaml)
+An example of configuring [prometheus.yaml](../prometheus/prometheus.yaml) for scraping FlashArray endpoints is here.
```

Comment:
[This is an example of configuring the prometheus.yaml.](../prometheus/prometheus.yaml)

Reply:
Done - thanks